src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -36,6 +36,8 @@ var HTMLDOMPropertyConfig = {
     allowFullScreen: HAS_BOOLEAN_VALUE,
     allowTransparency: 0,
     alt: 0,
+    // 'as' is keyword to links with 'preload' type
```

Comment:
One last thing: I think the grammar might be a little bit confusing as-is. Can we change this to:
```js
// specifies target context for links with `preload` type
```
That's more in line with how the spec describes it as well. After that I think we can merge 👍